cockatrice/src/dlg_creategame.cpp

```diff
@@ -205,7 +206,7 @@ void DlgCreateGame::actReset()
         gameTypeCheckBoxIterator.value()->setChecked(false);
     }
 
-descriptionEdit->setFocus();
```

Comment:
Had to add this back because actReset is also called when the clear button is clicked.